test/functional/world_location_news_controller_test.rb

```diff
@@ -98,37 +114,28 @@ def assert_featured_editions(editions)
     assert_equal "<#{@world_location.analytics_identifier}>", response.headers["X-Slimmer-World-Locations"]
   end
 
-  view_test "should show featured links if there are some" do
-    featured_link = create(:featured_link, linkable: @world_location)
-
-    get :index, world_location_id: @world_location
-
-    assert_select '.featured-links' do
-      assert_select "a[href='#{featured_link.url}']", text: featured_link.title
-    end
-  end
-
-  view_test "should redirect to the world location page if locale is not 'en'" do
-    LocalisedModel.new(@world_location, :fr).update_attributes(name: "Territoire antarctique britannique")
-    get :index, world_location_id: @world_location, locale: 'fr'
+  view_test "restricts atom feed entries to those with the current locale" do
+    world_location = create(:world_location, translated_into: [:fr])
 
-    assert_redirected_to world_location_path(@world_location)
-  end
+    translated_edition = create(:published_publication, world_locations: [world_location], translated_into: [:fr])
+    create(:published_publication, world_locations: [world_location])
 
-  view_test "should redirect to the world location page if the user is in 'A' cohort" do
-    with_variant WorldwidePublishingTaxonomy: "A", assert_meta_tag: false do
-      get :index, world_location_id: @world_location
+    get :index, world_location_id: world_location.id, format: :atom, locale: 'fr'
 
-      assert_redirected_to world_location_path(@world_location)
+    assert_select_atom_feed do
+      with_locale :fr do
+        assert_select_atom_entries([translated_edition])
```

Comment:
Exactly... I copied it from the [WorldLocationsController tests](https://github.com/alphagov/whitehall/blob/master/test/functional/world_locations_controller_test.rb#L322-L335) since this page is essentially a trimmed down copy of it